gems/pending/spec/util/postgres_admin_spec.rb

```diff
@@ -55,5 +55,23 @@
           .to eql "su - postgres -c '/ctl/path stop -W -D /pgdata/path -s -m fast'"
       end
     end
+
+    describe ".initialized?" do
+      around do |example|
+        Dir.mktmpdir do |dir|
+          ENV["APPLIANCE_PG_DATA"] = dir
```

Comment:
Ok, thanks, I didn't see the above context.  Let's come back to that in a future PR.  Polluting ENV in test can cause weird errors so we should try to rollback our changes to it.